__tests__/index.test.tsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable react/display-name */
```

Comment:
**Quick question**

What is this rule?

Reply:
I have some questions in this test that I want to review with you tomorrow :)

Reply:
This has been fixed!